upup/pkg/fi/vfs_castore.go

```diff
@@ -233,27 +233,27 @@ func (c *VFSCAStore) parseKeysetYaml(data []byte) (*kops.Keyset, error) {
 // loadCertificatesBundle loads a keyset from the path
 // Returns (nil, nil) if the file is not found
 // Bundles avoid the need for a list-files permission, which can be tricky on e.g. GCE
-func (c *VFSCAStore) loadKeysetBundle(p vfs.Path) (*keyset, error) {
+func (c *VFSCAStore) loadKeysetBundle(p vfs.Path) (*keyset, string, error) {
```

Comment:
You could put the stored version into the (internal) keyset type - should be simpler

Reply:
need more context